great_expectations/expectations/core/unexpected_rows_expectation.py

```diff
@@ -133,6 +140,37 @@ def _prescriptive_template(
         )
         return renderer_configuration
 
+    @classmethod
+    @renderer(renderer_type=LegacyRendererType.PRESCRIPTIVE)
+    @render_suite_parameter_string
+    @override
+    def _prescriptive_renderer(
+        cls,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        runtime_configuration: Optional[dict] = None,
+        **kwargs,
+    ) -> list[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+        params = substitute_none_for_missing(
+            configuration.kwargs,  # type: ignore[union-attr]
+            ["unexpected_rows_query"],
+        )
+
+        template_str = "Unexpected rows query: $unexpected_rows_query"
```

Comment:
How does this look when the query is multi-line? I suspect it's multi-line more often than not. Adding a newline is probably better IMO.

Reply:
Do you just mean a newline after the `:`?

Reply:
Yes that's what I mean